config/application.rb

```diff
@@ -272,6 +282,21 @@ def watchable_args
     Sprockets.register_mime_type 'application/javascript', extensions: ['.js', '.es6', '.js.es6'], charset: :unicode
     Sprockets.register_postprocessor 'application/javascript', DiscourseJsProcessor
 
+    # The sprockets sourcemap pipeline overrites our ember-cli-generated sourcemaps, so let's disable it
+    # (it doesn't work with our DiscourseJsProcessor anyway)
+    config.assets.configure do |env|
+      env.cache = nil # TODO REMOVE THIS
```

Comment:
I think it'll be helpful if we can elaborate further on the comment here and why there is a need to remove this.